src/storages/MongoDBStorage.js

```diff
@@ -0,0 +1,62 @@
+import _ from 'lodash';
+import Bluebird from 'bluebird';
+import Storage from './Storage';
+
+/**
+ * @class JSONStorage
+ */
+export default class MongoDBStorage extends Storage {
+    /**
+     * Constructs MongoDB file storage.
+     *
+     * @param {Object} [options]
+     * @param {String} [options.path='./umzug.json'] - Path to JSON file where
```

Comment:
Missing documentation for options.connection, options.collection, and options.collectionName.